### DISCUSSION TITLE
What is the roadmap around `serverless@2` (support, maintenance, EOL, etc).

### DISCUSSION THREAD

**Discussion by neverendingqs** [serverless/serverless#10934] [Q&A] [answered]
With the release of `serverless@3`, the number of `serverless@2` releases have drastically decreased (which makes sense).

How long should we expect `serverless@2` to be maintained, and when is the planned end-of-life date? Is there a page on https://www.serverless.com/ I should be looking at?

Thanks!

**Comment by mnapoli** [comment, marked as answer]
Hi @neverendingqs,

We don't plan on releasing new features or minor bug fixes in v2. If we discover a major issue in v2, we will fix it and tag new v2 releases. But all new features and minor fixes will land in v3.

Does that answer your question?
